src/ng/directive/ngOptions.js

```diff
@@ -303,6 +303,7 @@ var ngOptionsDirective = ['$compile', '$parse', function($compile, $parse) {
         values = values || [];
 
         Object.keys(values).forEach(function getWatchable(key) {
+          if (key.charAt(0) === '$') return;
```

Comment:
Good point. For some reason, [`ToJsonReplacer()`][1] was modifed some time ago to only ignore `$$` properties (but everything else (e.g. `ngRepeat`, `ngOptions`, `equals`) kept ignoring `$` as well. Too much breaking change I guess :)

So, it's fine as it is :smiley:

[1]: https://github.com/angular/angular.js/blob/d7dc14dc0cdeb9c187d227e19acc8aca7df9d740/src/Angular.js#L1102-L1116